packages/basic-dark/src/index.ts

```diff
@@ -41,6 +41,8 @@ export const basicDarkTheme = EditorView.theme(
     },
 
     '.cm-cursor, .cm-dropCursor': { borderLeftColor: cursor },
+    //Block cursor. Make sure to comment out or remove default cursor styles before using
```

Comment:
can you remove these comment lines?

Reply:
Done. Let me know if I missed anything. 

Thanks!